spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-config-sample/README.adoc

```diff
@@ -0,0 +1,78 @@
+= Spring Cloud GCP Pub/Sub Configuration Management Code Sample
+
+This code sample demonstrates automated configuration management using Spring Cloud Config, Spring Cloud Bus and GCP Pub/Sub.
+The sample app consists of a configuration server and a small web application that subscribes to the server to be notified of property changes.
+
+Configuration server runs on port `8888`, and can read configuration from a local filesystem (useful for exploring the sample) or from a source control system.
+When the server is notified that new configuration is available, it fetches the updated configuration and sends it out via Spring Cloud Bus.
+Under the hood, Spring Cloud Bus uses the Spring Cloud Stream binder for GCP Pub/Sub to send a message to a topic named `springCloudBus`.
+The client uses the same binder with an anonymous subscription to receive notifications, and provides an HTTP endpoint `/message` for printing the dynamically updated message value.
+
+== Setup
+
+1. Configure your GCP project ID and credentials by following link:../../docs/src/main/asciidoc/core.adoc#project-id[these instructions].
++
+Alternatively, if you have the https://cloud.google.com/sdk/[Google Cloud SDK] installed and initialized, and are logged in with https://developers.google.com/identity/protocols/application-default-credentials[application default credentials], Spring will auto-discover those parameters for you.
+
+2. Make sure that GCP Cloud Pub/Sub API is enabled.
+
+== Running the code sample with configuration derived from local filesystem
+
+1. In separate terminal windows, start the `spring-cloud-gcp-pubsub-config-sample-server` and `spring-cloud-gcp-pubsub-config-sample-client` applications with the `$ mvn spring-boot:run` command in the same folder as the apps' `pom.xml` files.
+
+2. Visit http://localhost:8080/message in a browser; observe the default value of `none` printed on the page.
```

Comment:
I assume this endpoint is provided by the client.
I think we need some more explanation of what the server is doing and what the client is doing.